microsoft-edge/webview2/concepts/print.md

```diff
@@ -0,0 +1,736 @@
+  return true;
+}
+```
+
+**APIs:**
+
+* [ICoreWebView2_16::ShowPrintUI(printDialogKind) Method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#showprintui)
+* [COREWEBVIEW2_PRINT_DIALOG_KIND Enum](/microsoft-edge/webview2/reference/win32/icorewebview2_16#corewebview2_print_dialog_kind)
+
+---
+
+<!-- end of tab-set -->
+
+
+<!-- ====================================================================== -->
+## Print
+
+The Print method silently print the current top-level document in the WebView2 using optional programmatically specified print settings. If you want to build your own print preview dialog or otherwise build your own print experience you can use this method. This API consists of an asynchronous Print method and a PrintSettings object.
+
+
+<!-- ------------------------------ -->
+
+# [C#](#tab/csharp)
+
+[CoreWebView2.PrintAsync() Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printasync#microsoft-web-webview2-core-corewebview2-printasync)
+[CoreWebView2PrintSettings Class](/dotnet/api/microsoft.web.webview2.core.corewebview2printsettings)
+
+
+<!-- ------------------------------ -->
+
+# [C++](#tab/cpp)
+
+[ICoreWebView2_16::Print() Method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#print)
+[ICoreWebView2PrintSettings2 Interface](microsoft-edge/webview2/reference/win32/icorewebview2printsettings2)
+[COREWEBVIEW2_PRINT_STATUS Enum](/microsoft-edge/webview2/reference/win32/icorewebview2_16#corewebview2_print_status)
+
+---
+
+<!-- end of tab-set -->
+
+### Example 1: Print to default printer
+
+This example prints the current web page without a print dialog to a default printer with the default print settings.
+
+
+<!-- ------------------------------ -->
+
+# [C#](#tab/csharp)
+
+```csharp
+async void PrintToDefaultPrinter ()
+{
+  string title = webView.CoreWebView2.DocumentTitle;
+  try
+  {
+    // Passing null for `PrintSettings` results in default print settings used.
+    // Prints current web page with the default page and printer settings.
+    CoreWebView2PrintStatus printStatus = await webView.CoreWebView2.PrintAsync(null);
+
+    if (printStatus == CoreWebView2PrintStatus.Succeeded)
+    {
+      MessageBox.Show(this, "Printing " + title + " document to printer is succeeded", "Print to default printer");
+    }
+    else if (printStatus == CoreWebView2PrintStatus.PrinterUnavailable)
+    {
+      MessageBox.Show(this, "Printer is not available, offline or error state", "Print to default printer");
+    }
+    else
+    {
+      MessageBox.Show(this, "Printing " + title + " document to printer is failed", "Print to default printer");
+    }
+  }
+  catch (Exception)
+  {
+    MessageBox.Show(this, "Printing " + title + " document already in progress", "Print to default printer");
+  }
+}
+```
+
+**APIs:**
+
+* [CoreWebView2.PrintAsync() Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printasync#microsoft-web-webview2-core-corewebview2-printasync)
+
```

Comment:
```suggestion
* [CoreWebView2PrintStatus Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2printstatus)

```
like in c++ below

https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2printstatus